contrib/agingBloom.h

```diff
@@ -28,59 +28,67 @@
 #include <time.h>
 #include <stdint.h>
 #include <stdbool.h>
+#include <math.h>
 
 #define CHECK_ALL 0
 
+//#define SLEEP 0.1     // Used only when static time is wanted between insertions
+                        // instead of real time. (defined in seconds)
+
 #define TWOHASH
 
 typedef struct ageBloom_s ageBloom_t;
-typedef uint64_t timestamp_t;
+typedef double_t timestamp_t;
```

Comment:
Can you check, @Nimbus79, if they can stay as uint64_t?

Reply:
I switched it to double so I could save the timestamp in seconds since I obtain it in microseconds. I assumed the user-specified time span was in seconds as well, so that's why I converted everything to seconds.